### PR TITLE
Group Bug Free - Code Review: Final report

### PR DESCRIPTION
**Pull request by Zcalory**
This week, we collect more paper based on the previous work and we start to do the data collection according to our review protocol.

### CODE REVIEW

File: 07-code-review.Rmd [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Code Review
 ## Review protocol
 This section describes the review protocol used for the systematic review presented in this
-section. The protcol has been set up using Kitchenham's method as described by Kitchenham et al.
+chapter. The protocol has been set up using Kitchenham's method as described by Kitchenham et al.
```
Comment (qurben) [inline 07-code-review.Rmd]: You can do
```
.. described by @kitchenham2007
```

to get 

```
... described by Kitchenham (2007)
```

File: 07-code-review.Rmd [hunk 2]
```diff
@@ -244,3 +353,10 @@ Reference: @xia2015should
 
 ### Automatically recommending peer reviewers in modern code review
 Reference: @zanjani2016automatically
+
+## Excluded papers
+The following papers have been excluded from the survey. These papers are candidates, but have not
+been added to the final survey for the stated reason.
+
+* [@cohen2010modern]: This book is not accessible via the TU Delft subscription of Safari Books
```
Comment (qurben) [inline 07-code-review.Rmd]: This does not properly compile:


You can use 
```
 * @cohen2010modern: This book... 
```
or
```
 * [@cohen2010modern] : This book...
```
(note the extra space)

Bookdown can be a bit finnicky with these kinds of things.

**Comment by qurben** [comment]
Please resolve the merge conflicts.

**Comment by Bjorn48** [comment]
Thanks for your comments @qurben! We have incorporated corresponding changes.

**Comment by Guillembonet** [comment]
I find the information very precise and organized. Also very useful summaries.

File: book.bib [hunk 3]
```diff
@@ -1493,75 +344,24 @@ @inproceedings{hassan2018hirebuild
   booktitle={Proceedings of the 40th International Conference on Software Engineering},
   pages={1078--1089},
   year={2018},
-  organization={ACM}
```
Comment (qurben) [inline book.bib]: Hey, these are references used in #19

**Comment by Inventitech** [comment]
Please indicate group name or other identifying information in PR title

File: book.bib (hunk 3 above)
Comment (qurben) [reply book.bib]: You are still removing references used in #19. Between the lines `%% BEGIN chart-increasing-keyboard-Analytics` and `%% END chart-increasing-keyboard-Analytics`

**Pull request merged**